src/main/java/uk/gov/hmcts/reform/lrdapi/domain/CourtVenueRequestParam.java

```diff
@@ -0,0 +1,22 @@
+package uk.gov.hmcts.reform.lrdapi.domain;
+
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
+
+@AllArgsConstructor
+@Getter
+@NoArgsConstructor
+@Setter
```

Comment:
Please check mutation coverage for this project, as annotations like this will require coverage

Reply:
Current coverage is at 95%: https://build.platform.hmcts.net/job/HMCTS_Nightly_RD/job/rd-location-ref-api/job/master/560/artifact/build/reports/pitest/index.html

Reply:
resolved